### PR TITLE
[KAN-46] Create unit tests for testing speed of program execution

### PR DESCRIPTION
**Pull request by SirFlarflajon**
Adding efficiency tests to run to make sure our code is running under 30 seconds with any given input, no matter the distance.

### CODE REVIEW

File: tests/notamTests.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
+from io import StringIO
 import unittest
+import warnings
 import notamFetch
 from NavigationTools import *
 from io import StringIO
```
Comment (JamesJWilliams3000) [inline tests/notamTests.py:6]: StringIO is imported twice.

File: tests/notamTests.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import time
 
 class TestNotams(unittest.TestCase) :
-
-    message_log = StringIO()
-
+    
+    fake_output = StringIO()
```
Comment (JamesJWilliams3000) [inline tests/notamTests.py:11]: Kinda nit-picky, but maybe we could rename it to `dummy_output`? Maybe put a comment explaining that the script requires an output object, but the tests don't need to access it, hence the dummy output.

**Review by JamesJWilliams3000 — CHANGES_REQUESTED** [review]

File: tests/notamTests.py (hunk 2 above)
Comment (SirFlarflajon) [reply tests/notamTests.py]: Yeah that makes sense, there is also a comment above the class that uses it more about why it's there but I can add one above the other class as well

File: tests/notamTests.py (hunk 1 above)
Comment (SirFlarflajon) [reply tests/notamTests.py:6]: Removed it, thanks!

**Review by Zombiepro89 — APPROVED** [review]

**Review by JamesJWilliams3000 — APPROVED** [review]

**Review by Zenpock — APPROVED** [review]

**Review by aidanperkins — APPROVED** [review]

**Pull request merged**